### ISSUE TITLE
Could not resolve archetype

### ISSUE THREAD

**Issue by pedrolucasoliva** [kolorobot/spring-mvc-quickstart-archetype#74]
I fallow all steps on STS with JDK 8 and external maven:

- Import archetype URI by Import ... > Projects from Git > Clone URI
- Install the archetype in local repository with mvn install
- Go to Preferences > Maven > Archetypes and Add Local Catalog
- Select the catalog from file (archetype-catalog.xml)
- Create new Maven project and select the archetype (remember so select Include snapshot archetypes)

But when i try create a project, i recive this error:

Could not resolve archetype com.github.spring-mvc-archetypes:spring-mvc-quickstart:1.0.0-SNAPSHOT from any of the configured repositories.
Could not resolve artifact com.github.spring-mvc-archetypes:spring-mvc-quickstart:pom:1.0.0-SNAPSHOT
Missing com.github.spring-mvc-archetypes:spring-mvc-quickstart:pom:1.0.0-SNAPSHOT

Anybody can help me? :)

**Comment by kolorobot** [comment]
To me it sounds like either local install was not done or include snapshot
archetypes was not selected.

Are you sure you have executed the steps as they are described?

2015-01-29 14:37 GMT+01:00 pedrolucasoliva <notifications@github.com>:

> I fallow all steps on STS with JDK 8 and external maven:
>
>    - Import archetype URI by Import ... > Projects from Git > Clone URI
>    - Install the archetype in local repository with mvn install
>    - Go to Preferences > Maven > Archetypes and Add Local Catalog
>    - Select the catalog from file (archetype-catalog.xml)
>    - Create new Maven project and select the archetype (remember so
>    select Include snapshot archetypes)
>
> But when i try create a project, i recive this error:
>
> Could not resolve archetype
> com.github.spring-mvc-archetypes:spring-mvc-quickstart:1.0.0-SNAPSHOT from
> any of the configured repositories.
> Could not resolve artifact
> com.github.spring-mvc-archetypes:spring-mvc-quickstart:pom:1.0.0-SNAPSHOT
> Missing
> com.github.spring-mvc-archetypes:spring-mvc-quickstart:pom:1.0.0-SNAPSHOT
>
> Anybody can help me? :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kolorobot/spring-mvc-quickstart-archetype/issues/74>.
>



-- 
Rafał.

skype:  kolorobot80
blog:     http://blog.codeleak.pl | http://blog.goyello.com
<http://blog.goyello.com/author/rborowiec/>
twitter:  http://twitter.com/#!/kolorobot <https://twitter.com/#!/kolorobot>
github:  http://github.com/kolorobot <https://github.com/kolorobot>

**Comment by pedrolucasoliva** [comment]
I did different and it works!
1 - New Project (on eclipse) > Maven > Check out Maven Project from SCM
2 - SCM url (SVN) i put the git rep: https://github.com/kolorobot/spring-mvc-quickstart-archetype.git
3 - Finish and select Maven Install on POM file.
After that is the same:
4 - Go to Preferences > Maven > Archetypes and Add Local Catalog
5 - Select the catalog from file (archetype-catalog.xml)
6 - Create new Maven project and select the archetype (remember so select Include snapshot archetypes)

Thanks kolorobot.

**Comment by chapita** [comment]
Hi kolorobot I had the same problem that pedrolucasoliva i followed all the steps, the test ran ok, also the build, but when i tried to create the new maven project the same error appeared. Any suggestion?
Running on STS 3.6.3.SR1, JDK 1.8 and MVN 3.2.5

Thanks in advanced

**Comment by kolorobot** [comment]
Hi chapita,

The problem is there is not enough time to sit down and release this to
maven central :) Then most problems described are gone.

In addition, I know people may be interested in generating project from
IDE, but I ALWAYS generate it from CLI and then import to IDE.


2015-03-14 4:33 GMT+01:00 chapita <notifications@github.com>:

> Hi kolorobot I had the same problem that pedrolucasoliva i followed all
> the steps, the test ran ok, also the build, but when i tried to create the
> new maven project the same error appeared. Any suggestion?
> Running on STS 3.6.3.SR1, JDK 1.8 and MVN 3.2.5
>
> Thanks in advanced
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kolorobot/spring-mvc-quickstart-archetype/issues/74#issuecomment-79787573>
> .
>



-- 
Rafał.

skype:  kolorobot80
blog:     http://blog.codeleak.pl | http://blog.goyello.com
<http://blog.goyello.com/author/rborowiec/>
twitter:  http://twitter.com/#!/kolorobot <https://twitter.com/#!/kolorobot>
github:  http://github.com/kolorobot <https://github.com/kolorobot>

**Comment by kolorobot** [comment]
I wonder if the version mismatch in my catalog.xml solves the issue: https://github.com/kolorobot/spring-mvc-quickstart-archetype/commit/4d607d209cd614eaa1929afd5c5189e9b6fe9e31

Could you please check. Thanks!

**Comment by ivanthemeh** [comment]
Kolorobot, I could use some help with an issue Im having. I keep getting a build failure even after following your instructions to the T, the error is a failed integration test.

**Comment by atillman34817** [comment]
We were able to resolve the issue by using a previous version of maven.

**Comment by willymays** [comment]
Kolorobot, I am using Maven 3.3.3.  I pulled the project from Github directly into Eclipse Luna.  When I run install on the pom I get: [ERROR] Failed to execute goal org.apache.maven.plugins:maven-archetype-plugin:2.2:integration-test (default-integration-test) on project spring-mvc-quickstart: Execution default-integration-test of goal org.apache.maven.plugins:maven-archetype-plugin:2.2:integration-test failed: ${maven.home} is not specified as a directory: 'C:\Developer\workspaces\spring-mvc-quickstart-archetype\EMBEDDED'. ->

I also tried maven 3.0.5 with the same results.

Any idea?
